TestFoundation/TestNSData.swift

```diff
@@ -103,7 +103,7 @@ class TestNSData: XCTestCase {
     
     func test_writeToURLOptions() {
         let saveData = try! Data(contentsOf: Bundle.main.url(forResource: "Test", withExtension: "plist")!)
-        let savePath = URL(fileURLWithPath: "/var/tmp/Test.plist")
+        let savePath = URL(fileURLWithPath: NSTemporaryDirectory()+"Test1.plist")
```

Comment:
Nit: Can you please put spaces before and after the operator? Also, why have you renamed the output file?

Reply:
Had to rename output file as bundle resources dir and tmp directory are the same the way the tests run on Android. Deleting Test.plist makes the next run of tests fail.